src/js/App/RootApp.js

```diff
@@ -17,6 +18,24 @@ import useOuiaTags from '../utils/useOuiaTags';
 import Banner from './Banners/Banner';
 import cookie from 'js-cookie';
 
+const QuickStartCatalog = lazy(() =>
+  import(/* webpackChunkName: "quick-start" */ './QuickStartCatalog').then(({ QuickStartCatalog }) => ({ default: QuickStartCatalog }))
+);
+
+const LazyQuickStartCatalog = ({ ...props }) => {
```

Comment:
Can we move this component to a standalone file. The RootApp is getting a bit cluttered.